channel_0:
  max_position: 7808
  max_us: 1952.0
  min_position: 4928
  min_us: 1232.0
  neutral_position: 6480
  neutral_us: 1620.0
channel_1:
  max_position: 8000
  max_us: 2000.0
  min_position: 3712
  min_us: 928.0
  neutral_position: 6007
  neutral_us: 1501.75
channel_10:
  max_position: 6400
  max_us: 1600.0
  min_position: 5696
  min_us: 1424.0
  neutral_position: 6000
  neutral_us: 1500.0
channel_11:
  max_position: 6912
  max_us: 1728.0
  min_position: 5120
  min_us: 1280.0
  neutral_position: 5524
  neutral_us: 1381.0
channel_2:
  max_position: 8000
  max_us: 2000.0
  min_position: 3712
  min_us: 928.0
  neutral_position: 7440
  neutral_us: 1860.0
channel_3:
  max_position: 6400
  max_us: 1600.0
  min_position: 2880
  min_us: 720.0
  neutral_position: 5626
  neutral_us: 1406.5
channel_4:
  max_position: 7232
  max_us: 1808.0
  min_position: 3840
  min_us: 960.0
  neutral_position: 6173
  neutral_us: 1543.25
channel_5:
  max_position: 6592
  max_us: 1648.0
  min_position: 4032
  min_us: 1008.0
  neutral_position: 5918
  neutral_us: 1479.5
channel_6:
  max_position: 5440
  max_us: 1360.0
  min_position: 4608
  min_us: 1152.0
  neutral_position: 5059
  neutral_us: 1264.75
channel_8:
  max_position: 6144
  max_us: 1536.0
  min_position: 5632
  min_us: 1408.0
  neutral_position: 6000
  neutral_us: 1500.0
channel_9:
  max_position: 6912
  max_us: 1728.0
  min_position: 5120
  min_us: 1280.0
  neutral_position: 6499
  neutral_us: 1624.75
